twitter-monitoring-relay.conf

```diff
@@ -0,0 +1,40 @@
+; Default Twitter statsite config for external Twitter customers
+; that are using Twitter Observability via our https endpoint.
+;
+[statsite]
+port = 8125
+udp_port = 8125
+log_level = INFO
+log_facility = local0
+flush_interval = 10
+timer_eps = 0.01
```

Comment:
Done.

On Mon, Nov 23, 2015 at 2:40 PM, Hao Huang <notifications@github.com> wrote:

> In twitter-monitoring-relay.conf
> <https://github.com/twitter-forks/statsite/pull/23#discussion_r45671941>:
>
> > @@ -0,0 +1,40 @@
> > +; Default Twitter statsite config for external Twitter customers
> > +; that are using Twitter Observability via our https endpoint.
> > +;
> > +[statsite]
> > +port = 8125
> > +udp_port = 8125
> > +log_level = INFO
> > +log_facility = local0
> > +flush_interval = 10
> > +timer_eps = 0.01
>
> I suggest 0.001 otherwise people can't get precise p999 value
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twitter-forks/statsite/pull/23/files#r45671941>.
>



-- 
@anthonyjasta